packages/components/inputs/async-creatable-select-input/src/async-creatable-select-input.js

```diff
@@ -372,6 +373,10 @@ AsyncCreatableSelectInput.propTypes = {
    * If cacheOptions is truthy, then the loaded data will be cached. The cache will remain until cacheOptions changes value.
    */
   cacheOptions: PropTypes.any,
+  /**
+   * When set to false, the select menu will not be closed after an option is selected. By default, it is set to true.
+   */
+  closeMenuOnSelect: PropTypes.bool,
```

Comment:
I think it would be great to also add the default value to the `defaultProps` attribute of the component:
```
AsyncCreatableSelectInput.defaultProps = {
  // Using "null" will ensure that the currently selected value disappears in
  // case "undefined" gets passed as the next value
  value: null,
  isSearchable: true,
  menuPortalZIndex: 1,
  closeMenuOnSelect: true,
};
```